dotnetv3/Lambda/scenarios/Lambda_Basics/Lambda_Basics/LambdaMethods.cs

```diff
@@ -0,0 +1,205 @@
+    /// Basics Scenario.
+    /// </summary>
+    public class LambdaMethods
+    {
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.DeleteLambdaFunction]
+
+        /// <summary>
+        /// Deletes a Lambda function.
+        /// </summary>
+        /// <param name="client">An initialized Lambda client object.</param>
+        /// <param name="functionName">The name of the Lambda function to
+        /// delete.</param>
+        /// <returns>A Boolean value that indicates where the function was
+        /// successfully deleted.</returns>
+        public async Task<bool> DeleteLambdaFunction(AmazonLambdaClient client, string functionName)
+        {
+            var request = new DeleteFunctionRequest
+            {
+                FunctionName = functionName,
+            };
+
+            var response = await client.DeleteFunctionAsync(request);
+
+            // A return value of NoContent means that the request was processed
+            // (in this case, the function was deleted, and the return value
+            // is intentionally blank.
+            return response.HttpStatusCode == System.Net.HttpStatusCode.NoContent;
+        }
+
+        // snippet-end:[lambda.dotnetv3.Lambda_Basics.DeleteLambdaFunction]
+
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.UpdateFunctionCode]
+
+        /// <summary>
+        /// Updates an existing Lambda function.
+        /// </summary>
+        /// <param name="client">An initialized Lambda client object.</param>
+        /// <param name="functionName">The name of the Lambda function to update.</param>
+        /// <param name="bucketName">The bucket where the zip file containing
+        /// the Lambda function code is stored.</param>
+        /// <param name="key">The key name of the source code file.</param>
+        /// <returns>A System Threading Task.</returns>
+        public async Task UpdateFunctionCode(
+            AmazonLambdaClient client,
+            string functionName,
+            string bucketName,
+            string key)
+        {
+            var functionCodeRequest = new UpdateFunctionCodeRequest
+            {
+                FunctionName = functionName,
+                Publish = true,
+                S3Bucket = bucketName,
+                S3Key = key,
+            };
+
+            var response = await client.UpdateFunctionCodeAsync(functionCodeRequest);
+            Console.WriteLine($"The Function was last modified at {response.LastModified}.");
+        }
+
+        // snippet-end:[lambda.dotnetv3.Lambda_Basics.UpdateFunctionCode]
+
+        // snippet-start:[lambda.dotnetv3.Lambda_Basics.InvokeFunction]
+
+        /// <summary>
+        /// Invokes a Lambda function.
+        /// </summary>
+        /// <param name="client">An initialized Lambda client object.</param>
+        /// <param name="functionName">The name of the Lambda function to
+        /// invoke.</param>
+        /// <returns>A System Threading Task.</returns>
```

Comment:
This one returns the result of the Lambda function, right? I think the return comment block is inaccurate if that is the case.

Reply:
Updated the return value description.